libs/ngx-mime/src/lib/viewer/viewer-footer/content-search-navigator/content-search-navigator.component.html

```diff
@@ -1,77 +1,71 @@
-<mat-toolbar class="content-search-navigator-toolbar" color="primary">
-  <div
-    *ngIf="searchResult"
-    fxLayout="row"
-    fxFlex
-    fxLayoutAlign="space-between center"
+<mat-toolbar
+  *ngIf="searchResult"
+  class="content-search-navigator-toolbar min-w-[270px]"
```

Comment:
Do we need to set min-width to 270px? seems to work as intended without

Reply:
Resolving this as it's added to prevent layoutshifting when using the pageslider